lib/moon/packages/debug/eventable.rb

```diff
@@ -2,12 +2,12 @@ module Debug
   module Eventable
     def self.pretty_print(obj, depth = 0)
       last_key = nil
-      obj.each_listener do |key, listener|
+      obj.each_listener do |key, cb|
```

Comment:
The module is pretty much broken and I refuse to fix it (again), so its being removed this time.